extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -118,32 +119,47 @@ interface ContextBase {
 	onNewInputComponentCreated: (name: string, inputComponentInfo: InputComponentInfo) => void;
 }
 
-export function createTextInput(view: azdata.ModelView, inputInfo: {
+function createInputBoxField({ context, isPasswordField = false, inputBoxType = 'text' }: { context: FieldContext; isPasswordField?: boolean; inputBoxType?: azdata.InputBoxInputType; }) {
```

Comment:
Renamed these because Label field map to the TextComponent fields in the core so use of 'Text' for these which map to InputBoxComponent in the core used to be a little confusing to me.

Reply:
The name: 'Field' are used for method that create a label+input field pair and the one without just create one of these items.

Reply:
you probably should add these comment directly to the code

Reply:
Done.